support/ebpf/tracemgmt.h

```diff
@@ -440,7 +440,9 @@ static inline EBPF_INLINE int get_next_interpreter(PerCPURecord *record)
   // Check if the section id happens to be in the interpreter map.
   OffsetRange *range = bpf_map_lookup_elem(&interpreter_offsets, &section_id);
   if (range != 0) {
-    if ((section_offset >= range->lower_offset) && (section_offset <= range->upper_offset)) {
+    if (
+      ((section_offset >= range->lower_offset1) && (section_offset <= range->upper_offset1)) ||
+      ((section_offset >= range->lower_offset2) && (section_offset <= range->upper_offset2))) {
```

Comment:
This adds ebpf code and makes it slower. And there's no users for this yet. Could this be commented out? Perhaps also split this out to a function to make this a bit more readable. Then the commented code could be in a `#if 0` instead.

Reply:
If you anticipate the follow up PR to happen soon, we can already leave this test enabled. Maybe still separate the test to a helper function?